_posts/2021-12-22-federated-learning-candig.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: blogpost
+title: Federated Learning on CanDIG
+summary: An introduction to federated learning and CanDIG-specific implementation considerations.
+image: img/posts/secure-cross-service-graphql-interface/architecture-upside-down.png
```

Comment:
Just double checking that this is the image you want to show up as the "unfurling image" - e.g. if you post this link in slack https://d4fcf33a.candig-github-io.pages.dev/posts/federated-learning-candig/ this will be the image that shows up